tests/views/test_reporting_units.py

```diff
@@ -7,26 +7,59 @@
 from response_operations_ui import app
 
 respondent_party_id = "cd592e0f-8d07-407b-b75d-e01fbdae8233"
+business_party_id = 'b3ba864b-7cbc-4f44-84fe-88dc018a1a4c'
+ru_ref = '50012345678'
+collection_exercise_id_1 = '14fb3e68-4dca-46db-bf49-04b84e07e77c'
+collection_exercise_id_2 = '9af403f8-5fc5-43b1-9fca-afbd9c65da5c'
+iac_1 = 'jkbvyklkwj88'
+iac_2 = 'ljbgg3kgstr4'
+survey_id = 'cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87'
 backstage_api_url = app.config["BACKSTAGE_API_URL"]
 CONNECTION_ERROR = 'Connection error'
 
-url_get_reporting_unit = f'{backstage_api_url}/v1/reporting-unit/50012345678'
-url_search_reporting_units = f'{backstage_api_url}/v1/reporting-unit/search'
 url_get_contact_details = f'{backstage_api_url}/v1/party/party-details?respondent_party_id={respondent_party_id}'
 url_edit_contact_details = f'{backstage_api_url}/v1/party/update-respondent-details/{respondent_party_id}'
 url_generate_new_code = f'{backstage_api_url}/v1/reporting-unit/iac/ce_id/ru_ref'
 url_resend_verification_email = f'{backstage_api_url}/v1/reporting-unit/resend-verification-email/{respondent_party_id}'
 url_change_enrolment_status = f'{app.config["BACKSTAGE_API_URL"]}/v1/party/change-enrolment-status'
 
-with open('tests/test_data/reporting_units/reporting_unit.json') as json_data:
-    reporting_unit = json.load(json_data)
+url_get_party_by_ru_ref = f'{app.config["PARTY_URL"]}/party-api/v1/parties/type/B/ref/{ru_ref}'
+url_get_cases_by_business_party_id = f'{app.config["CASE_URL"]}/cases/partyid/{business_party_id}'
+url_get_casegroups_by_business_party_id = f'{app.config["CASE_URL"]}/casegroups/partyid/{business_party_id}'
+url_get_collection_exercise_by_id = f'{app.config["COLLECTION_EXERCISE_URL"]}/collectionexercises'
+url_get_business_party_by_party_id = f'{app.config["PARTY_URL"]}/party-api/v1/businesses/id/{business_party_id}'
+url_get_available_case_group_statuses_direct = f'{app.config["CASE_URL"]}/casegroups/transitions' \
+                                               f'/{collection_exercise_id_1}/{ru_ref}'
+url_get_survey_by_id = f'{app.config["SURVEY_URL"]}/surveys/{survey_id}'
+url_get_respondent_party_by_party_id = f'{app.config["PARTY_URL"]}/party-api/v1/respondents/id/{respondent_party_id}'
+url_get_iac = f'{app.config["IAC_URL"]}/iacs'
+
 with open('tests/test_data/reporting_units/respondent.json') as json_data:
     respondent = json.load(json_data)
-with open('tests/test_data/reporting_units/edited_reporting_unit.json') as json_data:
-    edited_reporting_unit = json.load(json_data)
 with open('tests/test_data/case/case.json') as json_data:
     case = json.load(json_data)
 
+with open('tests/test_data/party/business_reporting_unit.json') as json_data:
```

Comment:
🎨  James (@JamesGardiner) made a good point about this in another [PR comment](https://github.com/ONSdigital/ras-frontstage/pull/328/files#r187295508): that this is not JSON but a handle to a file and should probably follow the convention of `fp` or similar.